packages/jest/src/resultsProcessor.ts

```diff
@@ -57,10 +57,11 @@ function processA11yErrors(testSuite: TestResult, testResult: AssertionResult) {
             // TODO (spike) : What happens if there are ever multiple failureDetails?
             //  Ideally there shouldn't be as test execution should be stopped on failure
             A11yResults.add(error.a11yResults, suiteName).forEach((a11yResult) => {
+                const className = `${suiteName}:[${a11yResult.wcag}]`;
```

Comment:
Is this the separator referenced from the `jest-tool` PR? If so, let's pull it out as a constant so it's more clear it has significance.

Reply:
Yes, just made those changes - pushing them now. Initially thought of exporting the const and importing it in jest-tool, but don't think the adding the import dependency overhead to jest-tool is justified in this case.